### DISCUSSION TITLE
A operational issues about DPGEN

### DISCUSSION THREAD

**Discussion by XQrobby** [deepmodeling/dpgen#1398] [Q&A]
### Summary

Hello developer. Does the Dpgen exploration phase support custom in. lammps files, as the work requires surveying some special MD processes, which require inserting some commands into in. lammps. If possible, please reply to the steps. Thank you.

### DP-GEN Version

all

### Details

The above in. lammps are the files required for MD when training the model with dpgen. I guess it should be possible to set in.lammps in param.json, or use a generator. Please reply to the developer.

**Comment by njzjz** [comment]
The path to the template file can be set in the [argument `run_jdata[model_devi_engine=lammps]/model_devi_jobs/template/lmp`](https://docs.deepmodeling.com/projects/dpgen/en/latest/run/param.html#argument:run_jdata[model_devi_engine=lammps]/model_devi_jobs/template/lmp).
